OpenRA.Mods.RA2/Traits/SpawnNeighboringActors.cs

```diff
@@ -0,0 +1,97 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
```

Comment:
Not entirely related to this PR in specific, but should copyright of non-engine/core code be attributed to engine/core?  Also copy+pasting `2007-<year>` around doesn't actually make that the case.

Reply:
I won't change just this file for consistency.